tachys/src/html/style.rs

```diff
@@ -327,23 +429,27 @@ impl IntoStyle for (Arc<str>, Arc<str>) {
         el: &crate::renderer::types::Element,
     ) -> Self::State {
         let style = Rndr::style(el);
-        (style, self.1)
+        (style, self.0, self.1)
     }
 
     fn build(self, el: &crate::renderer::types::Element) -> Self::State {
         let (name, value) = self;
         let style = Rndr::style(el);
         Rndr::set_css_property(&style, &name, &value);
-        (style, value)
+        (style, name, value)
     }
 
     fn rebuild(self, state: &mut Self::State) {
         let (name, value) = self;
-        let (style, prev) = state;
-        if value != *prev {
+        // state.1 was the previous name, theoretically the css name could be changed:
+        if name != state.1 {
```

Comment:
If this is the case, then we should also `*prev_name = name`, or it will stick with the old name if this updates again.